yarn plugin with run/link/unlink/test and standalone

giardb opened the pull request.

lorenzoalfieri commented:
The PR is working, but there is some improvement to perform:

1) The icon in a plan is a plain circle, you should try to make the cat contour transparent.
![yarn icon](https://user-images.githubusercontent.com/23459153/29481025-7184bfa0-847d-11e7-8585-88d5ed01adf0.png)

2) Diagnostics are all info. You should make a controller for each action to create error diagnostics and avoid useless ones.
3) A lot of output should not be diagnostics, like the loading bar.

giardb commented:
- changed the svg icon, it's now without the round background, only the cat.
- added a replace to erase some ANSI character.
- some basic detection for warning, success and error diagnostics.

alanzanattadev commented:
Why it modifies Jest and ESLint ? We also need to remove the import from "atom" because it's not compatible anymore with the new architecture
